src/compas_view2/objects/__init__.py

```diff
@@ -17,13 +18,15 @@
 from compas.geometry import Capsule
 from compas.geometry import Polyhedron
 
+from ..shapes import VectorGroup
 from ..shapes import Arrow
-from ..shapes import Arrow2d
 from ..shapes import Text
 from ..collections import Collection
 
 from .object import Object
 
+from .vectorobject import VectorObject
+from .vectorgroupobject import VectorGroupObject
```

Comment:
collection of vectors?

Reply:
Indeed, the slightly challenging part is that it uses a different shader, but shouldn't be too difficult to deal with tho